lib/src/components/NxLoadError/__tests__/NxLoadError.test.tsx

```diff
@@ -4,86 +4,99 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import NxLoadError, { Props } from '../NxLoadError';
-import NxButton from '../../NxButton/NxButton';
-import { NxErrorAlert } from '../../NxAlert/NxAlert';
+import { screen } from '@testing-library/react';
+import { rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import userEvent from '@testing-library/user-event';
+import NxLoadError from '../NxLoadError';
 
 describe('NxLoadError', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxLoadError, {});
+  const renderEl = rtlRenderElement(NxLoadError, {});
 
   it('does not render anything if error is unset', function() {
-    expect(getShallowComponent()).toBeEmptyRender();
+    expect(renderEl()).not.toBeInTheDocument();
   });
 
   it('renders an error alert when error is set', function() {
-    expect(getShallowComponent({ error: 'Error!' }))
-        .toContainMatchingElement(NxErrorAlert);
+    const el = renderEl({ error: 'Error!' });
+    expect(el).toHaveAttribute('role', 'alert');
   });
 
   it('passes a default title message if there is an error', function() {
-    expect(getShallowComponent({ error: 'Error!' }).render().text())
-        .toContain('An error occurred loading data');
+    const el = renderEl({ error: '' });
+    expect(el?.textContent).toContain('An error occurred loading data. ');
   });
 
   it('uses the specified title message instead of the default', function() {
-    expect(getShallowComponent({ error: 'Error!', titleMessage: 'This is bad' }).render().text())
-        .not.toContain('An error occurred loading data');
+    const el = renderEl({ error: '', titleMessage: 'This is bad!' });
+    expect(el?.textContent).not.toBe('An error occurred loading data. ');
+    expect(el?.textContent).toContain('This is bad! ');
 
-    expect(getShallowComponent({ error: 'Error!', titleMessage: 'This is bad' }).render().text())
-        .toContain('This is bad');
   });
 
   it('Uses the error as children', function() {
```

Comment:
"Children" isn't really a DOM concept, it's a react concept.
```suggestion
  it('renders the error as part of the text content', function() {
```